src/__test__/modal.test.js

```diff
@@ -1,42 +1,30 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
-import Modal  from '../components/Modal';
+import { shallow } from 'enzyme';
+import { Modal } from '../components/Modal';
```

Comment:
This line is wrong, I fixed it in a commit. The modal component is a default export so doesn't need the curly braces. In fact it comes out as undefined hence the error `TypeError: Cannot read property 'contextTypes' of undefined` when you run tests.